packages/hero/index.test.js

```diff
@@ -1,4 +1,23 @@
-// import Hero from './'
+import { render } from '@testing-library/react'
+import Hero from '.'
+import props from './props'
+import { getTestValues } from 'swingset/testing'
+
+const defaultProps = getTestValues(props)
+
+it('should add a provided className to the root element', () => {
```

Comment:
I added the `className` prop in this PR, so felt right to add a corresponding test.